public/calendar_reboot.js

```diff
@@ -207,6 +208,8 @@ CheckIt.prototype.onActivityChanged = function(storageObj) {
     // Passing storageObj as argument to have access to activity calls, which
```

Comment:
Just pass the activity level directly if you want to do this. If you really want to access the entire store, you could have gotten it via `this.store` anyway.